shell/platform/fuchsia/flutter/flatland_platform_view.cc

```diff
@@ -85,30 +87,31 @@ void FlatlandPlatformView::OnGetLayout(
 
   float pixel_ratio = view_pixel_ratio_ ? *view_pixel_ratio_ : 1.0f;
 
-  SetViewportMetrics({
-      pixel_ratio,  // device_pixel_ratio
-      std::round(view_logical_size_.value()[0] *
-                 pixel_ratio),  // physical_width
-      std::round(view_logical_size_.value()[1] *
-                 pixel_ratio),  // physical_height
-      0.0f,                     // physical_padding_top
-      0.0f,                     // physical_padding_right
-      0.0f,                     // physical_padding_bottom
-      0.0f,                     // physical_padding_left
-      0.0f,                     // physical_view_inset_top
-      0.0f,                     // physical_view_inset_right
-      0.0f,                     // physical_view_inset_bottom
-      0.0f,                     // physical_view_inset_left
-      0.0f,                     // p_physical_system_gesture_inset_top
-      0.0f,                     // p_physical_system_gesture_inset_right
-      0.0f,                     // p_physical_system_gesture_inset_bottom
-      0.0f,                     // p_physical_system_gesture_inset_left,
-      -1.0,                     // p_physical_touch_slop,
-      {},                       // p_physical_display_features_bounds
-      {},                       // p_physical_display_features_type
-      {},                       // p_physical_display_features_state
-      0,                        // p_display_id
-  });
+  SetViewportMetrics(kFlutterDefaultViewId,
+                     {
```

Comment:
I'd consider making a new variable for the metric. This formatting isn't ideal :/

Reply:
Same comment for `gfx_platform_view.cc`